### PR TITLE
Compass refact

### PR DESCRIPTION
**Pull request by gabrielleitezup**
## Issue Description

Compass will be the new core of Charles

### CODE REVIEW

File: compass/.env [hunk 1]
```diff
@@ -4,7 +4,21 @@ DB_HOST=localhost
...
 DB_PORT=5432
-ENV=PROD
 PLUGINS_DIR=./dist
+MOOVE_URL=http://charlescd-moove:8080
+DISPATCHER_INTERVAL=15s
+MOOVE_USER=admin@admin
+MOOVE_PATH=http://charlescd-moove:8080
+MOOVE_AUTH="Bearer 123"
+MOOVE_DB_USER=charles
+MOOVE_DB_PASSWORD=         charles
```
Comment (barbararochazup) [inline compass/.env:14]: blank spaces in moove_db_password

Comment (icaroafonsozup) [reply compass/.env]: Blank spaces removed.

File: compass/.env [hunk 2]
```diff
@@ -4,7 +4,21 @@ DB_HOST=localhost
...
+MOOVE_DB_USER=charles
+MOOVE_DB_PASSWORD=charles
+MOOVE_DB_HOST=localhost
+MOOVE_DB_NAME=charles
+MOOVE_DB_SSL=disable
+MOOVE_DB_PORT=5432
+REQUESTS_PER_SECOND_LIMIT= 1
+LIMITER_TOKEN_TTL=5
+LIMITER_HEADERS_TTL=5
+ENCRYPTION_KEY=caf5a807-5edd-4580-9149-7a4882755716
```
Comment (barbararochazup) [inline compass/.env:22]: this only to local purpose?

File: compass/cmd/db.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
+/*
+ *
+ *  Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
```
Comment (barbararochazup) [inline compass/cmd/db.go:3]: update header to have last year edition as Copyright 2020, 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA

File: compass/cmd/http.go [hunk 4]
```diff
@@ -0,0 +1,182 @@
+/*
+ *
+ *  Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
```
Comment (barbararochazup) [inline compass/cmd/http.go:3]: update header to have last year edition as Copyright 2020, 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA

File: compass/cmd/jobs.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
+/*
+ *
+ *  Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
```
Comment (barbararochazup) [inline compass/cmd/jobs.go:3]: update header to have last year edition as Copyright 2020, 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA

**Pull request closed without merging**